.circleci/config.yml

```diff
@@ -0,0 +1,191 @@
+            sh scripts/install-android-tools.sh
+            echo 'export PATH=$ANDROID_SDK_ROOT/tools/bin:$PATH'  >> $BASH_ENV
+            echo 'export PATH=$ANDROID_SDK_ROOT/tools:$PATH'  >> $BASH_ENV
+            echo 'export PATH=$ANDROID_SDK_ROOT/platform-tools:$PATH'  >> $BASH_ENV
+            echo 'export PATH=$ANDROID_SDK_ROOT/emulator:$PATH'  >> $BASH_ENV
+            source $BASH_ENV
+            sdkmanager --list
+      - save_cache:
+          key: android=tools-v1-{{ checksum "scripts/install-android-tools.sh" }}-{{ arch }}
+          paths:
+            - ~/android-tools
+
+  create-launch-android-emulator:
+    description: "create and launch android emulators"
+    steps:
+      - run:
+          name: create AVD
+          command: |
+            echo "no" | avdmanager --verbose create avd --force \
+            --name "Pixel_3a_API_29" \
+            --package "system-images;android-29;google_apis_playstore;x86_64"
+
+      - run:
+          name: start AVD
+          command: emulator @Pixel_3a_API_29 -no-window -no-audio
+          background: true
+
+      - run:
+          name: wait for emulator
+          command: |
+            adb wait-for-device shell 'while [[ -z $(getprop dev.bootcomplete) ]]; do sleep 1; done;'
+
+  setup-flutter:
+    description: "setup Flutter"
+    steps:
+      - run:
+          name: Download flutter SDK
+          command: git clone -b stable https://github.com/flutter/flutter.git flutter
+      - run:
+          name: Setup environment variables
+          command: echo 'export PATH="$PATH:`pwd`/flutter/bin"'  >> $BASH_ENV
+      - run:
+          name: Flutter doctor
+          command: flutter doctor
+
+jobs:
+  lint:
+    description: "Run static analysis for Flutter"
+    docker:
+      - image: cirrusci/flutter:stable
+    steps:
+      - checkout
+      - build-flutter-project
+      - run:
+          name: Analyze code
+          command: flutter analyze lib
+
+  test:
+    description: "Run tests for Flutter"
+    docker:
+      - image: cirrusci/flutter:stable
+    steps:
+      - checkout
+      - build-flutter-project
+      - run:
+          name: Run tests
+          command: flutter test
+
+  android-integration-test:
+    description: "Run integration tests for Flutter"
+    macos:
+      xcode: 11.3.1
+    steps:
+      - checkout
+      - setup-flutter
+      - android-sdk-dependencies
+      - create-launch-android-emulator
+      - build-flutter-project:
+          project: example
+      - run-flutter-integration-tests
+
+  ios-integration-test:
+    description: "Run integration tests for Flutter"
+    macos:
+      xcode: 11.3.1
+    steps:
+      - checkout
+      - setup-flutter
+      - restore_cache:
+          name: Restore cocoapods specs and pods
+          key: v1-cocoapods-{{ checksum "./example/ios/Podfile.lock" }}-{{ arch }}
+      - simulator-dependencies
+      - run:
+          name: Open simulator
+          command: xcrun simctl boot "iPhone 11"
```

Comment:
is this needed? 
Also, we might wanna pre-start it https://circleci.com/docs/2.0/testing-ios/#reducing-job-time-and-best-practises